Polyhedron/demo/Polyhedron/Scene_plane_item.h

```diff
@@ -133,11 +133,19 @@ public Q_SLOTS:
   }
   
   void setNormal(float x, float y, float z) {
-    frame->setOrientation(x, y, z, 0.f);
+    QVector3D normal(x,y,z);
+    QVector3D origin(0,0,1);
+    QQuaternion q(CGAL::sqrt((normal.lengthSquared()) * (origin.lengthSquared())) + QVector3D::dotProduct(origin, normal),QVector3D::crossProduct(origin, normal));
+    q.normalize();
+    frame->setOrientation(q.x(), q.y(), q.z(), q.scalar());
   }
 
   void setNormal(double x, double y, double z) {
-    frame->setOrientation((float)x, (float)y, (float)z, 0.f);
+    QVector3D normal(x,y,z);
+    QVector3D origin(0,0,1);
+    QQuaternion q(CGAL::sqrt((normal.lengthSquared()) * (origin.lengthSquared())) + QVector3D::dotProduct(origin, normal),QVector3D::crossProduct(origin, normal));
+    q.normalize();
+    frame->setOrientation(q.x(), q.y(), q.z(), q.scalar());
   }
```

Comment:
To avoid the copy-paste, `void setNormal(double x, double y, double z)` could be simplified to:
```C++
void setNormal(double x, double y, double z) {
  setNormal(float(x), float(y), float(z));
}
```